rero_ils/modules/loans/tasks.py

```diff
@@ -67,3 +70,29 @@ def cancel_expired_request_task(tstamp=None):
     set_timestamp('cancel-expired-request-task', total=total_loans_counter,
                   cancelled=total_cancelled_loans)
     return total_loans_counter, total_cancelled_loans
+
+
+@shared_task(ignore_result=True)
+def delete_loans_created(verbose=False, hours=1, dbcommit=True, delindex=True):
+    """Delete loans with state CREATED."""
+    now = datetime.now(timezone.utc)
+    if hours > 0:
+        now += timedelta(hours=hours)
```

Comment:
are you sure about the "+=" ? It should be "-=" isn't it ?

Reply:
You "resolved" but without answer. 
In my function comprehension (docstring is poor), you would delete all loans with status created AND creation date is from [* TO now-1h]. So for me your timedelta will be "-="

Reply:
Sorry, corrected it.